test/server/runner-test.js

```diff
@@ -325,4 +437,39 @@ describe('Runner', function () {
             testFilter(filter, expectedTestNames, done);
         });
     });
+
+    describe('.run()', function () {
```

Comment:
All bootstrapping-related tests should be in this section, because bootstrapping happens on `run`.